OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -84,8 +84,15 @@ public IEnumerable<IOrderTargeter> Orders
 
 		public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool queued)
 		{
-			if (order.OrderID == "PortableChronoDeploy" && CanTeleport)
-				self.World.OrderGenerator = new PortableChronoOrderGenerator(self, Info);
+			if (order.OrderID == "PortableChronoDeploy")
+			{
+				// HACK: Switch the global order generator instead of actually issuing an order
+				if (CanTeleport)
+					self.World.OrderGenerator = new PortableChronoOrderGenerator(self, Info);
+
+				// HACK: We need to issue a fake order to stop the game complaining about the bodge above
```

Comment:
Note that this is restoring and documenting the workaround that I removed in #13995 after assuming it unintentionally bad code.